R/dbplyr.R

```diff
@@ -0,0 +1,282 @@
+#' @include DBI.R
+NULL
+
+temporaryErrorMessage <- paste0("You cannot use `temporary = TRUE`",
+  "when using a Pool object, since temporary tables are local to a ",
+  "connection, and there's no guarantee you'll get the same ",
+  "connection back next time. You must either create a permanent ",
+  "table, or checkout a connection from `pool` directly with ",
+  "`con <- poolCheckout(pool)`, and then release the connection ",
+  "back to the pool when you're finished (`poolReturn(con)`).")
+
+# --- These generics are set in dplyr (not database-specific)
+#' @export
+copy_to.Pool <- function(dest, df, name = deparse(substitute(df)),
+  overwrite = FALSE, ...) {
+    db_con <- poolCheckout(dest)
```

Comment:
If it's quick to explain, can you elaborate on why, @jcheng5? Also, @hadley, this is merely in your head now, right? (I couldn't find any instances of `localPoolCheckout`in github)